### ISSUE TITLE
StarFinder conversion?

### ISSUE THREAD

**Issue by cinder-byte** [zombieFox/awesomeSheet#176]
Hello! I love using awesomeSheet since I came across it a few months back. My group is considering trying out StarFinder & PathFinder 2 later this year & I was wondering if you & your team have any plans for converting this to work for SF/PF2? Keep up the great work.

**Comment by zombieFox** [comment]
Hi, glad you like the project. There isn't a team, just myself developing the project alone. I have vague plans to look into supporting Pathfinder 2. However no work has begun yet. Mostly because I don't play Pathfinder 2 and don't know the system inside and out like (mostly) I do with Pathfinder 1. This might change once Pathfinder 2 is out and my group and I stat playing it. I don't plan to support Starfinder for similar reason, my group and I don't play it. 

In short, no work done or concrete plans right now. Maybe in the future.
